digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/TierekisteriTestApi.scala

```diff
@@ -237,6 +315,14 @@ class TierekisteriTestApi extends ScalatraServlet with JacksonJsonSupport {
       roadWidth
     } else if (fieldCode == trafficSignTRCode) {
       trafficSign
+    } else if (fieldCode == laneArragementTRCode) {
+      massTransitLane
+    } else if (fieldCode == pavedRoadTRCode) {
+      pavedRoad
+    } else if (fieldCode == damagedByThawTRCode) {
+      damagedByThaw
+    } else if (fieldCode == europeanRoadTRCode) {
+      europeanRoad
```

Comment:
Maybe we could map these, especially since we are adding more assets all the time? Same for the other requests